worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -145,6 +145,11 @@ private void handleSingleBatchId(final String batchId, final BatchWorkerServices
                 batchWorkerServices.registerBatchSubtask("subbatch:" + tenantId.getValue() + "/" + batchIdExtracted.getValue()
                     + "/" + subbatch.getName());
             }
+        } catch (final IllegalArgumentException ex) {
+            log.error("Exception while reading the batch: " + batchId
+                + ", it was not found");
```

Comment:
Remove "it" from both messages so it reads:
"Exception while reading the batch: " + batchId + " was not found"